packages/playwright-test/src/mount.ts

```diff
@@ -100,3 +108,28 @@ function wrapFunctions(object: any, page: Page, callbacks: Function[]) {
     }
   }
 }
+
+function contextHash(context: BrowserContextOptions): string {
+  const hash = {
+    acceptDownloads: context.acceptDownloads,
+    bypassCSP: context.bypassCSP,
+    colorScheme: context.colorScheme,
+    extraHTTPHeaders: context.extraHTTPHeaders,
+    forcedColors: context.forcedColors,
+    geolocation: context.geolocation,
+    hasTouch: context.hasTouch,
+    httpCredentials: context.httpCredentials,
+    ignoreHTTPSErrors: context.ignoreHTTPSErrors,
+    isMobile: context.isMobile,
+    javaScriptEnabled: context.javaScriptEnabled,
+    locale: context.locale,
+    offline: context.offline,
+    permissions: context.permissions,
+    proxy: context.proxy,
+    storageState: context.storageState,
+    timezoneId: context.timezoneId,
+    userAgent: context.userAgent,
+    deviceScaleFactor: context.deviceScaleFactor,
+  };
```

Comment:
I worry we will add a context option and never remember to add here making for a gnarly debug. (Especially given most of these are upstream browser options, they will rarely change and we will forget to edit here.)

Can we change this to be defensive so by default new context options will create a cache miss.

Something like:

```
const EXCLUDE_FROM_HASH = […];
const entriesToHash = Object.entries(context).filter(([key]) => !EXCLUDE_FROM_HASH.includes(key));
```